API.md

```diff
@@ -24,19 +24,19 @@ Constructs a new object of data bus software SPI with arguments:
 - `pinSCK`: pin SCK the clock pulses which synchronize data transmission generated by the master. Any GPIO-pin can be used.
 - `devicesCount`: quantity of octofets boards connected in one chain: logical output pin headers of one module to the inputs of another. If the `devicesCount` argument is omitted, it used only one board Octofet.
 
-Difference between hardware and software SPI is speed–with. Software SPI it will run much slower than hardware SPI, but hardware SPI works only on specific pins: `MOSI`, `MISO`(not used in Octofet) and `SCK`. Use software SPI only when hardware SPI pins are used by other devices.
+Difference between hardware and software SPI is speed. Software SPI it will run much slower than hardware SPI, but hardware SPI works only on specific pins: `MOSI`, `MISO`(not used in Octofet) and `SCK`. Use software SPI only when hardware SPI pins are used by other devices.
 
 ### `void begin(uint8_t* resetValues = NULL)`
 
-Initializes the given interface, prepares the board for communication. The argument `resetValues` states power switches of all devices by default as 8-bit value. One bit for one power switch: 1 is turn-on and 0 is turn-off. If `resetValues` argument is adjusted, then when calling method `reset()`, power switches sets value by `resetValues`. If `resetValues` argument is omitted — power switches sets is turn-off.
+Initializes the given interface, prepares the board for communication. The argument `resetValues` states power switches of all devices by default as 8-bit value. One bit for one power switch: 1 is turn-on and 0 is turn-off. If `resetValues` argument is adjusted, then when calling method `reset()`, power switches sets value by `resetValues`. If `resetValues` argument is omitted — power switches are set to turn-off state.
```

Comment:
1) power switches sets value by `resetValues`. -> power switches are set by `resetValues`.
2) If `resetValues` argument is omitted — power switches are set to turn-off state.  ->  If `resetValues` argument is omitted, power switches are set to turn-off state.